win32ss/user/ntuser/msgqueue.c

```diff
@@ -2530,6 +2530,20 @@ MsqSetStateWindow(PTHREADINFO pti, ULONG Type, HWND hWnd)
    return NULL;
 }
 
+VOID FASTCALL
+MsqReleaseModifierKeys(PUSER_MESSAGE_QUEUE MessageQueue)
+{
+    WORD ModifierKeys[] = { VK_LCONTROL, VK_RCONTROL, VK_CONTROL,
+                            VK_LMENU,    VK_RMENU,    VK_MENU,
+                            VK_LSHIFT,   VK_RSHIFT,   VK_SHIFT };
+
+    for (UINT i = 0; i < ARRAYSIZE(ModifierKeys); ++i)
```

Comment:
```suggestion
    for (UINT i = 0; i < _countof(ModifierKeys); ++i)
```
Nitpick and optional: If it builds like this as well, then you can save 1 character by using _countof. The one character is my reason to usually favor it over ARRAYSIZE, except when working within specific Wine-code which would already heavily use the ARRAYSIZE-macro.